pkg/reconciler/subscription/subscription.go

```diff
@@ -139,6 +139,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, key string) error {
 func (r *Reconciler) reconcile(ctx context.Context, subscription *v1alpha1.Subscription) error {
 	subscription.Status.InitializeConditions()
 
+	if err := subscription.Validate(ctx); err != nil {
+		// Verify subscription is valid
```

Comment:
Can you move this comment to the other side of the if statement and end it with a `.`